components/generic/LanguageSelector.vue

```diff
@@ -71,7 +71,7 @@
 
   .language-icon {
     top: 7px;
-    width: 18px;
+    width: 31px;
     height: 28px;
```

Comment:
The height needs a few extra pixels too. There's an arrow under the icon that's currently just cut off.